### DISCUSSION TITLE
Console version of Librehwmonitor

### DISCUSSION THREAD

**Discussion by rtomik** [LibreHardwareMonitor/LibreHardwareMonitor#751] [Q&A]
Hi where can i download latest console/cmd version of hw monitor? I couldn't find it anywhere. I dont have visual studio. I have older version of Librehardwaremonitorreport.exe  but it doesn't support new CPU's

**Comment by bpsib** [comment]
+1. I also need a command line version of this as OpenHardwareMonitor doesn't work on my PC. Can anyone find or make a cmd version of LibreHardwareMonitor?
